buildlib/dockers/ubuntu-release.Dockerfile

```diff
@@ -14,13 +16,15 @@ RUN apt-get update && \
         libcap2 \
         libnuma-dev \
         libtool \
+        libnvidia-compute-${DRIVER_VERSION} \
         make \
         maven \
         udev \
         wget \
         environment-modules \
         pkg-config \
-    && apt-get remove -y openjdk-11-* || apt-get autoremove -y \
+        sudo \
```

Comment:
Unneeded anymore?

Reply:
Replaced by a longer remove command.

Reply:
I mean we don't need to instal sudo anymore

Reply:
Correct, yet we may keep it for future needs.

Reply:
Agreed upon resolving it for now.